include/circt/Dialect/SV/SVDPI.td

```diff
@@ -0,0 +1,73 @@
+//===- SVDPI.td - SV DPI Interface Ops ---------------------*- tablegen -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This describes the ops for SystemVerilog verification statements and
+// declarations.
+//
+//===----------------------------------------------------------------------===//
+
+include "mlir/IR/FunctionInterfaces.td"
+
+def DPIImportOp : SVOp<"dpi.import", [
+        FunctionOpInterface,
+        Symbol,
+        HasParent<"mlir::ModuleOp">]> {
+  let summary = "Import a DPI-C function";
+  let description = [{
+    Represents a SV DPI-C import statement. Imports a C function for use.
+
+    ```
+    import "DPI-C" function void some_function(
+      input  bit[31:0] value_in,
+      output bit[31:0] value_out
+    )
+    ```
+  }];
+
+  // TODO: Add support for the `pure` attribute.
+  // TODO: Add support for return values.
+
+  let arguments = (ins TypeAttrOf<FunctionType>:$function_type,
```

Comment:
import allows separate verilog v.s. C name (optionally).  We may want to think about this early too.

Reply:
I think this is best handled in a subsequent PR.